src/libraries/Microsoft.NETCore.Platforms/src/runtimeGroups.props

```diff
@@ -99,7 +99,19 @@
     <RuntimeGroup Include="iossimulator">
       <Parent>ios</Parent>
       <Architectures>arm64;x64</Architectures>
-      <Versions>10;11;12;13;14</Versions>
+      <Versions>10;11;12;13;14;15</Versions>
+    </RuntimeGroup>
+
+    <RuntimeGroup Include="tvos">
+      <Parent>unix</Parent>
+      <Architectures>arm64;x64</Architectures>
```

Comment:
Hmm, should this have x64?

Reply:
good catch, this is a leftover from when we didn't have the ios/iossimulator split, will update.

Reply:
hmm it runs into the following build errors:
```
runtimeGroups.props(277,5): error : Runtime maccatalyst-x64 imports ios-x64 which is not defined.
runtimeGroups.props(277,5): error : Runtime iossimulator-x86 imports ios-x86 which is not defined. 
runtimeGroups.props(277,5): error : Runtime iossimulator-x64 imports ios-x64 which is not defined. 
runtimeGroups.props(277,5): error : Runtime tvossimulator-x64 imports tvos-x64 which is not defined.
```

I don't think we can fix that if we still want to keep the iossimulator -> ios fallback, which I think we do?

Reply:
I don't understand the error. Why does it matter that there is no tvos-x64?

Reply:
hmm - I think we need tvos-x64 simply to preserve the relationship for tvossimulator-x64 (and others).

Reply:
Why? I think we want to block people building packages which target tvos-x64 or not?

Reply:
I'm not sure there's any alternative to preserve the api inheritance.  

@ericstj do you happen to know?

Reply:
yeah I think what we want is an `tvossimulator-x64` -> `tvos` fallback instead of a `tvos-x64` one, but from a quick look it seems the runtime graph generation logic is implemented in NuGet so not easy to change here.